tests/e2e/config/playwright.config.js

```diff
@@ -0,0 +1,69 @@
+const { defineConfig, devices } = require( '@playwright/test' );
+const { url } = require( './default.json' );
+
+module.exports = defineConfig( {
+	testDir: '../specs',
+
+	/* Maximum time one test can run for. */
+	timeout: 120 * 1000,
+
+	expect: {
+		/**
+		 * Maximum time expect() should wait for the condition to be met.
+		 * For example in `await expect(locator).toHaveText();`
+		 */
+		timeout: 20 * 1000,
+	},
+
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !! process.env.CI,
```

Comment:
💅  I don't understand well this comment.

I will suggest the one appearing below on the web:
https://playwright.dev/docs/test-configuration#:~:text=Whether%20to%20exit%20with%20an%20error%20if%20any%20tests%20are%20marked%20as%20test.only.%20Useful%20on%20CI.

Whether to exit with an error if any tests are marked as "test.only"

Reply:
Thanks, this is mostly copying and pasting from other examples. I think we can just remove this setting all together.